src/views/world/view_world.cpp

```diff
@@ -1,34 +1,26 @@
-#include "views/view.hpp"
 #include "util/entity.hpp"
-#include "util/ped.hpp"
 #include "util/notify.hpp"
+#include "util/ped.hpp"
 #include "util/vehicle.hpp"
+#include "views/view.hpp"
 
 namespace big
 {
-    void view::world()
-    {
-        components::sub_title("GUI_TAB_TIME_N_WEATHER"_T);
-        {
-            view::time_and_weather();
-        }
-
-        ImGui::Separator();
-        
-        components::sub_title("GUI_TAB_WATER"_T);
-        {
-		    components::command_checkbox<"partwater">();
-        }
```

Comment:
Where did this go to?

Reply:
Its a feature that does not work, said so in the command itself. Looked like waste of space so I removed it.

Reply:
Makes sense.